include/devicetree.h

```diff
@@ -145,6 +145,15 @@
  */
 #define DT_NODELABEL(label) DT_CAT(DT_N_NODELABEL_, label)
 
+/**
+ * @brief returns list of child node identifiers.
+ * List contains only praimary enabled child nodes (no aliases).
+ *
+ * @param path parent node path
+ * @return list of child node identifiers under path.
+ */
+#define DT_CHILD_NODELABELS(path) DT_CAT(path, _CHILD_NODELABELS)
```

Comment:
I think DT_CHILDREN() would be a better name. These are not node labels, they are node identifiers, and we don't want to mix those two concepts up in the API names.

Reply:
OK

Reply:
It would also be nice to see this placed after DT_PARENT() and DT_CHILD() in devicetree.h.

Reply:
ok